### PR TITLE
Tfsec fixes

### PR DESCRIPTION
**Pull request by anero**
:canned_food: 
- Removed OpenVPN module that we no longer use.
- Skip IAM policy validations on permissions that don't support specifying resources

### CODE REVIEW

File: iam_policy_describe_ec2/main.tf [hunk 1]
```diff
@@ -13,6 +13,8 @@ locals {
 
 resource "aws_iam_policy" "describe_ec2" {
   name = "${var.app}${local.region_prefix}-${var.env}-describe-ec2"
+  # Currently EC2 Describe* API doesn't support resource-level permissions (https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-policy-structure.html#UsingWithEC2_Actions)
```
Comment (lavaturtle) [inline iam_policy_describe_ec2/main.tf]: Would swapping out `ec2:Describe*` for more specific actions let us avoid the wildcard resource?

File: openvpn/main.tf [hunk 2]
```diff
@@ -1,239 +0,0 @@
-#--------------------------------------------------------------
-# This module creates all resources necessary for OpenVPN
```
Comment (lavaturtle) [inline openvpn/main.tf:2]: :broom:

**Review by lavaturtle — APPROVED** [review]
:+1:

File: iam_policy_describe_ec2/main.tf (hunk 1 above)
Comment (anero) [reply iam_policy_describe_ec2/main.tf]: Good question! Sadly `autoscaling:DescribeAutoScalingInstances` and `autoscaling:DescribeAutoScalingGroups` also don't support specifying resources: https://docs.aws.amazon.com/autoscaling/ec2/userguide/control-access-using-iam.html#policy-auto-scaling-resources. Will add to the comment to clarify.

**Pull request merged**